modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx

```diff
@@ -85,7 +85,7 @@ export function FieldBase({
 					<ClayTooltipProvider>
 						<span data-tooltip-align="top" title={tooltip}>
 							<ClayIcon
-								className="lfr-objects__field-base-tooltip-icon"
```

Comment:
Any idea why this sort of class name existed in the first place? Why the `lfr-objects__` part?

Reply:
I suppose that it is because the original component belongs to the Objects module. They use the `lfr-objects__` prefix for some components but not for others :smile: . 

x.e. the RichTextLocalized has a `lfr-notification__` prefix.

Reply:
Gotcha, so you just removed it because of re-locating the file? (btw I have no problem changing it, I just don't know why the objects team would prefix like that, thought maybe something related to tests?)

Reply:
Yes, I rename it because of re-location, as the component will be a general purpose one.